packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1239,6 +1249,8 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     _selectionOverlay = null;
     _focusAttachment.detach();
     widget.focusNode.removeListener(_handleFocusChanged);
+    WidgetsBinding.instance.removeObserver(this);
+    _clipboardStatus.removeListener(_onChangedClipboardStatus);
```

Comment:
dispose the _clipboardStatus since you own it?

Reply:
👍 See https://github.com/flutter/flutter/pull/54902#discussion_r418316612